src/Esia/Config.php

```diff
@@ -66,6 +70,7 @@ public function __construct(array $config = [])
         if (!$this->certPath) {
             throw new InvalidConfigurationException('Please provide certPath');
         }
+        $this->signerClass = $config['signerClass'] ?? '\Esia\Signer\SignerPKCS7';
```

Comment:
Don't need to specify class in the config if we use it as a dependency for `OpenId` class

Reply:
@fr05t1k May I ask you to show what do you mean in code?

Reply:
```
$config = new Config();
$openId = new OpenId($config);
$openId->setSigner(new CliSignerPKCS7());
```
In this case we can avoid dynamic class creation and allow to config Signer separately.